test/unit/client-config-configmap.bats

```diff
@@ -61,24 +61,11 @@ load _helpers
   [ "${actual}" = "true" ]
 }
 
-#--------------------------------------------------------------------
-# connectInject.healthChecks
-
-@test "client/ConfigMap: check_update_interval is not set by default" {
-  cd `chart_dir`
-  local actual=$(helm template \
-      -s templates/client-config-configmap.yaml  \
-      . | tee /dev/stderr |
-      yq '.data["config.json"] | length > 0' | tee /dev/stderr)
-  [ "${actual}" = "false" ]
-}
-
-@test "client/ConfigMap: check_update_interval is set when health checks enabled" {
+@test "client/ConfigMap: check_update_interval is set when connect is enabled" {
```

Comment:
Now that health checks are "always on" we will do this anytime connect is enabled.